src/Components/PostCard.js

```diff
@@ -0,0 +1,16 @@
+import data from '../Services/static-posts.json';
+import { CardAdresse, CardArticle, CardItem } from './StyledPostCard';
+
+export default function Posts() {
+	return (
+		<CardArticle>
+			<CardItem>
+				<h3>{data[0].name}</h3>
```

Comment:
headings shall be in order
(only one h1 per page, from there on h2 -> h3 -> ...)
you are using the h3 and h5 for styling, which is not commanded. use span instead and style it to your favor.

Reply:
Thanks for you feedback. I have changes my PR.